Makefile

```diff
@@ -302,7 +302,8 @@ ci-runner-github-tests:\
 	ci-job-capsules\
 	ci-job-chips\
 	ci-job-tools\
-	ci-job-miri
+	ci-job-miri\
+	ci-job-tests
```

Comment:
If I understand correctly, this is just a compilation test right?

I think I should probably go under the `github-build` in that case (it's similar in spirit to the `syntax` or `compilation` run).

`-tests` is pretty generic. Can we rename to `ci-job-qemu-compile-only` or something like that?

Reply:
Yeah, it's just a compilation test of the `cargo test`. I'll move the section

It's not a qemu test, so I don't think that's the right name. I'll change it to `ci-job-cargo-test`